app/code/Magento/InventoryShipping/view/adminhtml/web/js/form/element/sourceCode.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Can be removed after https://github.com/magento-engcom/msi/pull/788/files#r178954281